src/components/ImageView/RegionView/InvariantShapes.tsx

```diff
@@ -89,7 +90,7 @@ export const Anchor = (props: AnchorProps) => {
     const handleCircleDraw = (ctx, shape) => {
         const reverseScale = 1 / shape.getStage().scaleX();
         const radius = CIRCLE_ANCHOR_RADIUS * reverseScale;
-        const offsetY = -ROTATOR_ANCHOR_HEIGHT / shape.getStage().scaleX();
+        const offsetY = props.isLineRegion ? 0 : -ROTATOR_ANCHOR_HEIGHT * reverseScale;
```

Comment:
should be `inverseScale` instead of `reverseScale`

Reply:
changed. also renamed other variables in `InvariantShapes.tsx`